activities/communication/communications-plan.md

```diff
@@ -0,0 +1,179 @@
+---
+type: resource
+--
+
+# Communications plan
+
+## Principles
+
+Our communications need to reflect our core values: open, quality, trust, community, helpful. They must also be non-partisan, in line with our [founding principles](https://about.publiccode.net/organization/mission.html#founding-principles).
+
+Our messages are positive and inspirational. Our community won’t be sustainable if we attract people motivated by fear or anger.
+
+Our communications are in plain English, but we link to highly relevant pages in other languages, for example about our members’ work.
+
+Our communications should show to showcase the Foundation as:
+
+* active, dynamic, getting things done
+* smart enough to be trusted with the importance and complexity of its work
+* plugged into and supportive of the network of visionaries and activists who also want this future
+* open, both about its intentions and operations (transparency) as well as learnings, failures and successes
+
+We believe it’s more important to talk concretely about what we’ve already done than what we hope to do.
+
+## Mission statement
+
+To enable public-purpose software and policy that is open and collaborative.
+
+## Business objectives
+
+1. Get more members ([in phase 1](https://about.publiccode.net/organization/roadmap.html))
+2. Boost public codebases under Foundation stewardship
+3. Build our codebase community and attract more contributors
+4. Get more contributors and funding for the Foundation
+5. Boost the public code ecosystem
+
+## Audience
+
+Our audience is divided into:
+
+* current Foundation users
+* future Foundation users
```

Comment:
Even though it has some content below, I think 'user' really needs some sort of definition here, as it feels quite out of place.

Reply:
This use is defo out of sync with #603.